addons/sale_project/models/project.py

```diff
@@ -94,20 +94,34 @@ def _compute_has_any_so_to_invoice(self):
     def _compute_sale_order_count(self):
         sale_order_items_per_project_id = self._fetch_sale_order_items_per_project_id({'project.task': [('is_closed', '=', False)]})
         for project in self:
-            sale_order_lines = sale_order_items_per_project_id.get(project.id, self.env['sale.order.line'])
+            sale_order_lines = sale_order_items_per_project_id.get(project.id, self.env['sale.order.line']).filtered(lambda sol: sol.company_id in self.env.companies)
             project.sale_order_line_count = len(sale_order_lines)
             project.sale_order_count = len(sale_order_lines.order_id)
 
-    def _compute_invoice_count(self):
+    def _get_move_counts(self, move_types):
         data = self.env['account.move.line']._read_group(
-            [('move_id.move_type', 'in', ['out_invoice', 'out_refund']), ('analytic_distribution', 'in', self.analytic_account_id.ids)],
+            [
+                ('move_id.move_type', 'in', move_types),
+                ('analytic_distribution', 'in', self.analytic_account_id.ids),
+                ('company_id', 'in', self.env.companies.ids),
+            ],
             groupby=['analytic_distribution'],
             aggregates=['move_id:count_distinct'],
         )
-        data = {int(account_id): move_count for account_id, move_count in data}
+        return {int(account_id): move_count for account_id, move_count in data}
```

Comment:
Why do we cast as int?

Reply:
I've checked and `account_id` seems to be a string that's why :)